packages/synapse-react-client/src/components/FeaturedDataTabs/FacetPlotsCard.tsx

```diff
@@ -70,7 +120,7 @@ const FacetPlotsCard: React.FunctionComponent<FacetPlotsCardProps> = ({
       const getColumnType = (
         facetToPlot: FacetColumnResult,
       ): ColumnTypeEnum | undefined =>
-        data?.columnModels?.find(
+        data?.columnModels.find(
```

Comment:
as a result of specifying the generic above, we know `columnModels` is defined if `data` is defined, so no `?` is needed.